mypy/stubtest.py

```diff
@@ -1133,6 +1140,16 @@ def parse_options(args: List[str]) -> argparse.Namespace:
         action="store_true",
         help="Ignore unused whitelist entries",
     )
+    config_group = parser.add_argument_group(
+        title='mypy config file',
+        description="Use a config file instead of command line arguments. "
+                    "Plugins and mypy path are the only supported "
+                    "configurations.",
+    )
+    config_group.add_argument(
+        '--mypy-config-file',
+        help="Configuration file, must have a [mypy] section",
+    )
```

Comment:
minor: a wording suggestion
```suggestion
    config_group.add_argument(
        '--mypy-config-file',
        help="An existing mypy configuration file, currently used by stubtest to help determine mypy path and plugins",
    )
```